### PR TITLE
added table online share report

### PR DESCRIPTION
**Pull request by himanshu-2764**

### CODE REVIEW

File: src/modules/database/entities/online-share-report.entity.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  reportLink: string;
+
+  @Column()
+  prescriptions: string;
+
+  @Column()
+  sharedOption: string;
+
+  @Column({ nullable: true })
+  sharedEmailIds: string;
```
Comment (vansham69420) [inline src/modules/database/entities/online-share-report.entity.ts:24]: I feel this should be an `Array`, no?

File: src/modules/database/entities/online-share-report.entity.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  emailId: string;
+
+  @Column()
+  reportId: string;
+
+  @Column()
+  reportLink: string;
+
+  @Column()
+  prescriptions: string;
```
Comment (vansham69420) [inline src/modules/database/entities/online-share-report.entity.ts:18]: Do we need a foreign key with the `PrescriptionPlan` table here?

File: src/modules/database/entities/online-share-report.entity.ts (hunk 1 above)
Comment (himanshu-2764) [reply src/modules/database/entities/online-share-report.entity.ts:24]: @vansham69420  I have discussed this with @RambhauWaykar . As the data will be queried by a non-technical individual, using a comma-separated format would be preferable over an array or JSON.

Comment (vansham69420) [reply src/modules/database/entities/online-share-report.entity.ts:24]: Ok cool!

File: src/modules/database/entities/online-share-report.entity.ts (hunk 2 above)
Comment (himanshu-2764) [reply src/modules/database/entities/online-share-report.entity.ts:18]: Its just need string data in that. and nothing else we need that.

**Review by prashantSyngenta — APPROVED** [review]

File: src/modules/database/entities/online-share-report.entity.ts (hunk 2 above)
Comment (vansham69420) [reply src/modules/database/entities/online-share-report.entity.ts:18]: Ok!

**Review by vansham69420 — APPROVED** [review]
LG

**Pull request merged**